### ISSUE TITLE
Espeak for Android: application speech rate overrides system settings

### ISSUE THREAD

**Issue by codeofdusk** [espeak-ng/espeak-ng#969]
I've built the latest commit (ab11439) for Android. In the @rhdunn Google Play version, the Espeak speech rate scaled the system rate, such that TalkBack could dynamically adjust the rate and rates well in excess of 450 WPM were possible. This latest build, however, forces Espeak's rate to the value set in Espeak preferences.

An option should be added to "force speaking rate to Espeak preference" to use this new behaviour (perhaps enable by default, but allow a user option for the system to control the rate as with 1.49). Failing that, the 1.49 behaviour should be restored.

**Comment by jaacoppi** [comment]
Can you do a git bisect to see which commit changes this behavior?

**Comment by codeofdusk** [comment]
No, because the old commits (1.48.x) require old Java versions that are no longer in Debian. The play store version has not been updated in a very long while!
